client/components/tinymce/plugins/media/advanced/index.jsx

```diff
@@ -70,8 +70,9 @@ export default function( editor ) {
 		onPostRender() {
 			this.innerHtml(
 				ReactDomServer.renderToStaticMarkup(
+					// eslint-disable-next-line jsx-a11y/no-interactive-element-to-noninteractive-role
 					<button type="button" role="presentation" tabIndex="-1">
```

Comment:
Removing the `role="presentation"` attribute looks like a better way to fix the a11y warning. I'm wondering why @aduth added it (long time ago in #2085) in the first place.

Reply:
Fixed!

Reply:
> Removing the `role="presentation"` attribute looks like a better way to fix the a11y warning. I'm wondering why @aduth added it (long time ago in #2085) in the first place.

If I had to guess, it was likely to mimic markup otherwise rendered by a TinyMCE button in this context.

In retrospect, I ought to have made this more clear.